packages/ember-views/lib/views/view.js

```diff
@@ -1305,6 +1305,10 @@ var View = CoreView.extend(
     }
 
     this._super.apply(this, arguments);
+
+    if (!this._viewRegistry) {
```

Comment:
Those tests should either then not require global Ember.View.views or need to mock it/require it. We can leave that for a later refactor though